modules/sdk-coin-avaxp/src/lib/exportTxBuilder.ts

```diff
@@ -0,0 +1,112 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BuildTransactionError, NotSupported, TransactionType } from '@bitgo/sdk-core';
+import { AtomicTransactionBuilder } from './atomicTransactionBuilder';
+import {
+  BaseTx,
+  ExportTx,
+  PlatformVMConstants,
+  SECPTransferOutput,
+  TransferableOutput,
+  Tx,
+  UnsignedTx,
+} from 'avalanche/dist/apis/platformvm';
+import { BN } from 'avalanche';
+import { AmountOutput } from 'avalanche/dist/apis/evm/outputs';
+
+export class ExportTxBuilder extends AtomicTransactionBuilder {
+  private _amount: BN;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  protected get transactionType(): TransactionType {
+    return TransactionType.Export;
+  }
+
+  /**
+   *
+   * @param value
+   */
+  amount(value: BN | string): this {
+    const valueBN = BN.isBN(value) ? value : new BN(value);
+    this.validateAmount(valueBN);
+    this._amount = valueBN;
+    return this;
+  }
+
+  initBuilder(tx: Tx): this {
+    super.initBuilder(tx);
+    const baseTx: BaseTx = tx.getUnsignedTx().getTransaction();
+    if (!this.verifyTxType(baseTx)) {
+      throw new NotSupported('Transaction cannot be parsed or has an unsupported transaction type');
+    }
+    const outputs = baseTx.getExportOutputs();
+    if (outputs.length != 1) {
+      throw new BuildTransactionError('Transaction can have one external output');
+    }
+    const output = outputs[0];
+    if (!output.getAssetID().equals(this.transaction._assetId)) {
+      throw new Error('AssetID are not equals');
+    }
+    const secpOut = output.getOutput();
+    this.transaction._locktime = secpOut.getLocktime();
+    this.transaction._threshold = secpOut.getThreshold();
+    this.transaction._fromAddresses = secpOut.getAddresses();
+    this._externalChainId = baseTx.getDestinationChain();
+    this._amount = (baseTx.getExportOutputs()[0].getOutput() as any as AmountOutput).getAmount();
+    this.transaction._utxos = this.recoverUtxos(baseTx.getIns());
+    return this;
+  }
+
+  static verifyTxType(baseTx: BaseTx): baseTx is ExportTx {
+    return baseTx.getTypeID() === PlatformVMConstants.EXPORTTX;
+  }
+
+  verifyTxType(baseTx: BaseTx): baseTx is ExportTx {
+    return ExportTxBuilder.verifyTxType(baseTx);
+  }
```

Comment:
do we need `public` and `static` definitions for `verifyTxType()`?
I think we can keep the `static` one